test/integration/quota-test.js

```diff
@@ -0,0 +1,46 @@
+var expect = require('chai').expect
+const getQuota = require('../../lib/utils').getQuota
+const overQuota = require('../../lib/utils').overQuota
+const path = require('path')
+const read = require('../utils').read
+const root = 'accounts-acl/config/templates/new-account/'
+// const $rdf = require('rdflib')
+
+describe('Get Quota', function () {
+  var prefs = read(path.join(root, 'settings/serverSide.ttl'))
+  it('Check that the file is readable and has predicate', function () {
```

Comment:
typical style would be lowercase and conjugated:
```
it('checks…
```
such that it reads "getQuota checks that the file…".

Reply:
good point.